curator/README.md

```diff
@@ -1,36 +1,38 @@
 # Curator
 
-Curator allows the admin to remove old indices from Elasticsearch on a per-project
-basis.  The pod will read its configuration from a mounted yaml file that
-is structured like this:
+[Curator](https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/about.html) helps you curate, or manage, your Elasticsearch indices and snapshots. This document describes how it's used in OpenShift Origin Aggregated Logging.
 
-    PROJECT_NAME:
-      ACTION:
-        UNIT: VALUE
+The pod runs periodically as a kubernetes cronjob. The time when the job gets started is configurable during [installation](https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_logging#optional-vars) or by changing environment variables of the cronjob. When the pod is started, it will read its configuration from two mounted yaml files:
+* [Configuration file](https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/configfile.html) - general config, e.g. elasticsearch hostname, path to certificates etc.
+* [OpenShift custom config](#openshift-custom-config) - a simple way how to remove old indices. A curator [actions file](https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/actionfile.html) is generated based on this config
 
-    PROJECT_NAME:
-      ACTION:
-        UNIT: VALUE          
-     ...      
+Optionally you can use the [actions file](https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/actionfile.html) directly - this allows you to access all of curator features, should you need to run them periodically. However this can lead to destructive results and thus is not recommended. If you want to use this, see [Using actions file](#using-actions-file)
```

Comment:
Yes, literally in the config map. The installer creates an empty file named actions.yaml.
Users can then just copy paste their actions file there.
By default our config is used.

Can you help with the wording? We should make it obvious that we don't want people to use the raw actions files and if they choose to use them, they will have to make sure that they don't delete internal indices by mistake.

Reply:
Sure, I would also mention the name of the file in the config map then.

Maybe phrasing could be something like - "Optionally you can use the actions file directly. This allows you to use any action that Curator has available to it to be run periodically. However, this is only recommended for advanced users as using this can be destructive to the cluster and can cause removal of required indices/settings from Elasticsearch. To learn more about using the actions file see using actions file".